app/src/main/java/nerd/tuxmobil/fahrplan/congress/details/EventDetailFragment.java

```diff
@@ -155,15 +155,23 @@ public void onViewCreated(@NonNull View view, @Nullable Bundle savedInstanceStat
             // Detailbar
 
             TextView t;
-            t = view.findViewById(R.id.lecture_detailbar_date_time_location);
+            t = view.findViewById(R.id.lecture_detailbar_date_time);
             if (lecture != null && lecture.dateUTC > 0) {
                 DateFormat df = SimpleDateFormat
                         .getDateTimeInstance(SimpleDateFormat.SHORT, SimpleDateFormat.SHORT);
-                t.setText(df.format(new Date(lecture.dateUTC)) + " - " + room);
+                t.setText(df.format(new Date(lecture.dateUTC)));
             } else {
                 t.setText("");
             }
 
+            t = view.findViewById(R.id.lecture_detailbar_location);
+            if (t != null) {
+                t.setText(room);
+            } else {
+                t.setText("");
+            }
+
+
```

Comment:
Please remove this empty line.

Reply:
I did the refactor in this commit 4f6abfc